dv/settings.py

```diff
@@ -181,4 +185,15 @@
     },
 }
 
+# Caching system
+
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.filebased.FileBasedCache',
+        'LOCATION': '/var/tmp/django_cache',
```

Comment:
manage.py import and manage.py import_news should clear this dir

Reply:
Caching cleared.